app/scripts/satisfaction.js

```diff
@@ -1,11 +1,22 @@
-window.onload = function () {
+$(document).ready(function() {
```

Comment:
$.ready and window.load are events that can fire at different times when the page is loading. Are you sure you won't run into a race condition? It might make sense just to put everything in $.ready. Also you can use ```$(function () {...});``` as a short hand to ready.